core/server/api_container/server/startosis_engine/startosis_validator/docker_images_validator.go

```diff
@@ -63,7 +63,8 @@ func fetchImageFromBackend(ctx context.Context, wg *sync.WaitGroup, imageCurrent
 	err := (*backend).FetchImage(ctx, image)
 	if err != nil {
 		logrus.Warnf("Container image '%s' download failed. Error was: '%s'", image, err.Error())
-		pullErrors <- startosis_errors.NewValidationError("Failed fetching the required image '%v', make sure that the image exists and is public", image)
+		pullErrors <- startosis_errors.NewValidationError("Failed fetching the required image '%v' due to the following error: %v\n", image, err.Error())
+		return
```

Comment:
we will fail on the first failure instead of validating all, is that what you intend?

Reply:
oh makes sense